bin/start

```diff
@@ -8,6 +8,9 @@ while ! mysqladmin ping -h"app-db" --silent; do
     echo -n "."
 done
 echo "Starting testmine, visit: http://192.168.99.100:3002/ in your browser."
+gem install bundler
```

Comment:
This feels a little out of place

Reply:
you are right, i have removed it and verified it works without it.